README.md

```diff
@@ -3,35 +3,27 @@
 Building and running manually/locally
 -------------------------------------
 
-Initialize your local postgresql database as described in datamodel/README.md, then
-```
-export db_hostname=localhost
-export DCSA_CORE_Version=0.7.11 #or whatever version is the right one
-```
-If running without auth0, disable it with
-```
-export AUTH0_ENABLED=false
-```
-Then build and run with
+1) Initialize your local postgresql database as described in [datamodel/README.md](https://github.com/dcsaorg/DCSA-Information-Model/blob/master/README.md), then
+
+2) Build **DCSA-Core** as described in [DCSA-Core/README.md](https://github.com/dcsaorg/DCSA-Core/blob/master/README.md), then
+
+3) Build **DCSA-Event-Core** as described in [DCSA-Event-Core/README.md](https://github.com/dcsaorg/DCSA-Event-Core/blob/master/README.md), then
```

Comment:
The user can down load these via maven if they have a github access token without having to build these themselves.